dbbackup/tests/test_connectors/test_postgresql.py

```diff
@@ -1,15 +1,13 @@
 from __future__ import unicode_literals
 
-from mock import patch
+from io import BytesIO
+
 from django.test import TestCase
-from six import BytesIO
+from mock import patch
 
-from dbbackup.db.postgresql import (
-    PgDumpConnector,
-    PgDumpGisConnector,
-    PgDumpBinaryConnector
-)
 from dbbackup.db.exceptions import DumpError
+from dbbackup.db.postgresql import (PgDumpBinaryConnector, PgDumpConnector,
+                                    PgDumpGisConnector)
```

Comment:
Do you think you could keep the format as before as that is easier to read?

```suggestion
from dbbackup.db.postgresql import (
    PgDumpBinaryConnector,
    PgDumpConnector,
    PgDumpGisConnector
)
```